download.md

```diff
@@ -9,7 +9,7 @@ no-wrapper: true
 <div class="row margin-bottom-xl">
     <div class="col-sm-12">
         <p class="lead">
-            <span class="glyphicon glyphicon-download"/>
+            <span class="glyphicon glyphicon-download"></span>
```

Comment:
Makes sense when you think about what a span is. I guess htmlproofer is just not a very good validator. I've found other things that are obviously incorrect html that it has missed.